cmd/kubeadm/app/constants/constants.go

```diff
@@ -46,8 +46,12 @@ const (
 	FrontProxyClientCertName           = "front-proxy-client.crt"
 	FrontProxyClientKeyName            = "front-proxy-client.key"
 
-	AdminKubeConfigFileName   = "admin.conf"
-	KubeletKubeConfigFileName = "kubelet.conf"
+	AdminKubeConfigFileName             = "admin.conf"
```

Comment:
`kubeadm alpha phase kubeconfig token --client-name foo --api-server https://ip-here --token bar` will give you that nearly that. It will output the kubeconfig to stdout with the token, but it's very easy to just remove the bar token afterwards.

We might consider a `kubeadm alpha phase kubeconfig basic` or `bare` or `server-only` or something if there's need